incubator/kafka/templates/statefulset.yaml

```diff
@@ -83,6 +93,15 @@ spec:
         volumeMounts:
         - name: datadir
           mountPath: "{{ .Values.dataDirectory }}"
+{{- if .Values.prometheusExport.enabled }}
+      - name: {{ template "kafka.name" . }}-prom-exporter
+        image: {{ .Values.prometheusExport.image }}:{{ .Values.prometheusExport.imageTag }}
+        args:
+        - --kafka.server=localhost:9092
+        ports:
+        - containerPort: 9308
```

Comment:
From looking at several other helms with prometheus exporters, it seems common practice to add the port to the corresponding service as well.

Reply:
hmm looking at few charts (https://github.com/kubernetes/charts/tree/b44304bc3d1eb4efb496cf82b71e84da1970e604/stable/redis and https://github.com/kubernetes/charts/tree/79feb6ca8f162e0a1d6bd7171f3a2d754923a1df/stable/postgresql) that pod doesn't have prometheus annotation, only service. I'll do the same (remove on pod and add it to service).